### DISCUSSION TITLE
Labeling in Multiple Control Plane in Single Cluster

### DISCUSSION THREAD

**Discussion by oversampling** [istio/istio#52870] [Q&A] [answered]
Hi, currently I faced issue in a Multiple control plane scenario where I reference this [doc](https://istio.io/latest/docs/setup/install/multiple-controlplanes/). I realize that you will have to label the namespace to tell which istiod will the workload under the namespace will be part of. However, in this case, does it mean that I could relabel the namespace to be part of another namespace. In this case, is the community recommended that I will have to create RBAC to restrict the relabel action from user or Istio have something like namespace selector to tell istiod which part of namespace it should take care. I think Openshift has something like [Service Mesh Member Roll](https://docs.openshift.com/container-platform/4.9/service_mesh/v2x/ossm-create-mesh.html#ossm-member-roll-modify-cli_ossm-create-mesh), but not sure is there something like this in open sources version.

**Comment by howardjohn** [comment, marked as answer]
Yes, for a multi-tenant setup like this you will want to restrict access to the Namespace object. How organizations typically do this is to provision namespaces for tenants, then give the tenants permissions only to objects within those namespaces (not including the Namespace object itself).
